.pytool/CISettings.py

```diff
@@ -167,7 +167,15 @@ def GetDependencies(self):
 
     def GetPackagesPath(self):
         ''' Return a list of workspace relative paths that should be mapped as edk2 PackagesPath '''
-        result = ["Platforms", "MU_BASECORE", "Common/MU", "Common/MU_TIANO", "Common/MU_OEM_SAMPLE"]
+
+        # Include all submodule paths
+        result = []
+        for submodule in self.GetRequiredSubmodules():
+            result.append(submodule.path)
+
+        # Include platform packages
+        result.append("Platforms")
```

Comment:
You could just initialize the array with Platforms `result = ["Platforms"]`

Reply:
I considered that but went with appending everything, so all items are treated similarly.

I don't have a personal preference and I'm happy to go with it being initialized.

Reply:
Done in PR commit d691bf77ec8d5fa28f36d658d7b1c6d081c9c42d